src/EditorFeatures/TestUtilities2/Intellisense/TestStateFactory.vb

```diff
@@ -27,6 +28,11 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.IntelliSense
             testState.Workspace.SetOptions(
                 testState.Workspace.Options.WithChangedOption(CompletionOptions.TriggerInArgumentLists, LanguageNames.CSharp, showCompletionInArgumentLists))
 
+            If ensureCompeltionProvidersAvailable Then
```

Comment:
"Compeltion"

Reply:
(many places this is mispelled).

Reply:
i think this flag should go away.  providers should always be available.

Reply:
Sure